modules/virt-supported-tekton-tasks.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
+
+:_content-type: REFERENCE
+[id="virt-supported-tekton-tasks_{context}"]
+= Virtual machine tasks supported by the Tekton Tasks Operator
+
+The following table shows the cluster tasks that are included as part of the Tekton Tasks Operator.
+
+.Virtual machine tasks supported by the Tekton Tasks Operator
+[cols="1,1",options="header"]
+|===
+| Task | Description
+
+| `create-vm-from-template`
+| Create a virtual machine from a template.
+
+| `copy-template`
+| Copy a virtual machine template.
+
+| `modify-vm-template`
+| Modify a virtual machine template.
+
+| `modify-data-object`
+| Create or delete data volumes or data sources.
+
+| `cleanup-vm`
+| Execute a script or a command in a virtual machine and stop or delete the virtual machine afterwards.
+
+| `disk-virt-customize`
+| Use the `virt-customize` tool to run a customization script on a target PVC.
+
+| `disk-virt-sysprep`
+| Use the `virt-sysprep` tool to run a sysprep script on a target PVC.
+
+| `execute-virt-sysprep`
```

Comment:
This is not correct name of the task.
The name should be: 
| `cleanup-vm`
| Execute commands and/or stop/delete VMs

Reply:
you are right. Then this task can be deleted, because this task doesn't exist